docs/data/scheduler/primitives/DayGridPrimitives.js

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { DateTime } from 'luxon';
+import { DayGrid } from '@mui/x-scheduler/primitives/day-grid';
+import { useWeekList } from '@mui/x-scheduler/primitives/use-week-list';
+import { useDayList } from '@mui/x-scheduler/primitives/use-day-list';
+import classes from './DayGridPrimitives.module.css';
+
+const events = [
+  {
+    id: '1',
+    start: DateTime.fromISO('2025-05-26T07:30:00'),
+    end: DateTime.fromISO('2025-05-26T08:15:00'),
+    title: 'Footing',
+    resource: 'personal',
+  },
+  {
+    id: '2',
+    start: DateTime.fromISO('2025-05-26T16:00:00'),
+    end: DateTime.fromISO('2025-05-26T17:00:00'),
+    title: 'Weekly',
+    resource: 'work',
+  },
+  {
+    id: '3',
+    start: DateTime.fromISO('2025-05-27T10:00:00'),
+    end: DateTime.fromISO('2025-05-27T11:00:00'),
+    title: 'Backlog grooming',
+    resource: 'work',
+  },
+  {
+    id: '4',
+    start: DateTime.fromISO('2025-05-27T19:00:00'),
+    end: DateTime.fromISO('2025-05-27T22:00:00'),
+    title: 'Pizza party',
+    resource: 'personal',
+  },
+  {
+    id: '5',
+    start: DateTime.fromISO('2025-05-28T08:00:00'),
+    end: DateTime.fromISO('2025-05-28T17:00:00'),
+    title: 'Scheduler deep dive',
+    resource: 'work',
+  },
+  {
+    id: '6',
+    start: DateTime.fromISO('2025-05-29T07:30:00'),
+    end: DateTime.fromISO('2025-05-29T08:15:00'),
+    title: 'Footing',
+    resource: 'personal',
+  },
+  {
+    id: '7',
+    start: DateTime.fromISO('2025-05-30T15:00:00'),
+    end: DateTime.fromISO('2025-05-30T15:45:00'),
+    title: 'Retrospective',
+    resource: 'work',
+  },
+];
+
+export default function DayGridPrimitives() {
+  const getWeekList = useWeekList();
+  const getDayList = useDayList();
+
+  const weeks = React.useMemo(() => {
+    const weeksFirstDays = getWeekList({
+      date: events[0].start.startOf('month'),
+      amount: 'end-of-month',
+    });
+
+    const tempWeeks = [];
+    for (let i = 0; i < weeksFirstDays.length; i += 1) {
+      const weekStart = weeksFirstDays[i];
+      const weekDays = getDayList({ date: weekStart, amount: 7 });
+      const weekDaysWithEvents = weekDays.map((date) => ({
+        date,
+        events: events.filter(
+          (event) =>
+            event.start.hasSame(date, 'day') || event.end.hasSame(date, 'day'),
```

Comment:
Nit: I don't think that in this scenario we should care about end dates. This filter should only be for placing the event in the right cells, but if you have an event that has a start date of 27.05 and an end date of 28.05, the same event shows up in both cells 🤔 
Super nit: ordering the vents by start time could be a nice